src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import supabase, { Schema } from "@/supabase";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+
+interface Props {
+    text: string;
+    date: Date;
+    location: string;
+}
+
+export type ParcelOfSpecificDateAndLocation = {
+    collection_datetime: Schema["parcels"]["collection_datetime"];
+    clients: Pick<
+        Schema["clients"],
+        "flagged_for_attention" | "full_name" | "address_postcode" | "delivery_instructions"
+    > | null;
+};
+
+const getParcelsOfSpecificDateAndLocation = async (
+    date: Date,
+    collectionCentre: string
+): Promise<ParcelOfSpecificDateAndLocation[]> => {
```

Comment:
This name seems a bit long and verbose, would `SpecificParcel` not do?

Reply:
Specific parcels doesn't tell us that it is specific based on date and location (i.e. the filter criteria)
Also it is an internal function not a reusable component so it should be fine to be lengthy but clear about what's its function?

Reply:
Sure if you think it is better that is fine!